src/js/options.js

```diff
@@ -127,7 +127,7 @@ function loadOptions() {
 
       // checks for firefox and chrome values to see if webrtc ip leak is protected from something other than pb
       if (result.value.endsWith("public_interface_only") || result.value.endsWith("non_proxied_udp")) {
-        $("#toggle_webrtc_mode").prop("checked", true;
+        $("#toggle_webrtc_mode").prop("checked", true);
```

Comment:
Do you have linting set up locally?

Reply:
I do, but it's not automatic. I'll set it up to be automatic on every save -- typically I give a look over `git diff` and then `make runff` to make sure it runs smoothly, then finally a run over with `make lint`. Having it be automatic on save will definitely prevent future versions of whatever mistake this was